gulpfile.js

```diff
@@ -89,6 +89,10 @@
 
     gulp.task(
         'package:clean',
+        [
+            'qa',
```

Comment:
P.S. I also tested each empirically just to be sure, and got the results described above.